### PR TITLE
Python Push-ups Part 1

### PR DESCRIPTION
**Pull request by Charlie-England**

### CODE REVIEW

**Comment by Charlie-England** [comment]
Adding Lesson 1 warmup problem solution and adding error testing

File: students/charlie_england/session01/break_me.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    #parsing errors
+    #fr x in range(1,10):
+        #print(x)
+    pass
+
+def attribute_error():
+    #trying to use an attribute (function) of an object when the function does not exist
+    integer = 8
+    integer.append(1) #cannot append to an integeter, can append to a list
+
```
Comment (PythonCHB) [inline students/charlie_england/session01/break_me.py:25]: perfect.

File: students/charlie_england/session01/lesson1.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+def sleep_in(weekday, vacation):
+    if weekday and not vacation:
+        return False
+    if not weekday:
```
Comment (PythonCHB) [inline students/charlie_england/session01/lesson1.py:4]: if an "if" and "else" both do the same thing -- do you need it at all?

**Review by PythonCHB — APPROVED** [review]

File: students/charlie_england/session01/lesson1.py (hunk 2 above)
Comment (Charlie-England) [reply students/charlie_england/session01/lesson1.py:4]: Yep, you don't need both. That was a silly mistake. Also, I think this was an older solution that I did on that account. this is a better one.

def sleep_in(weekday, vacation):
  if weekday and not vacation:
    return False
  else:
    return True

**Pull request merged**